### ISSUE TITLE
helicopter verticallly takeoff and landing if possible

### ISSUE THREAD

**Issue by ylZvky** [dcs-liberation/dcs_liberation#395]
Currently helicopter  will taxi to runway then takeoff from runway even it is possilbe to takeoff vertically from parking position which would save the runway resource for fixed-wing vehicles.

**Comment by Moguy164** [comment]
i think that's a dcs limitation

**Comment by DanAlbert** [comment]
Helicopters that are parked in a parking space will always do that, yeah. With some effort I think we could claim the parking spaces but use takeoff from ground to avoid it.

**Comment by Khopa** [comment]
@DanAlbert 
I think the best thing we could do, is to add additional FARP slots in the campaign miz files for each airfield and prioritize those for helos.

**Comment by DanAlbert** [comment]
Once this is done we should remember to allow helos/harriers to use damaged runways.
